cross/pthread-stubs/Makefile

```diff
@@ -0,0 +1,38 @@
+PKG_NAME = libpthread-stubs
+PKG_VERS = 0.4
+PKG_EXT = tar.bz2
+PKG_DIST_NAME = $(PKG_NAME)-$(PKG_VERS).$(PKG_EXT)
+PKG_DIST_SITE = http://www.intel.com
+PKG_DIR = $(PKG_NAME)-$(PKG_VERS)
+
+DEPENDS =
+
+HOMEPAGE = http://www.intel.com
+COMMENT  = 
+LICENSE  = GPL
+
+GNU_CONFIGURE = 1
+
+YASM_PATH = $(WORK_DIR)/../../../native/yasm/work-native/install/usr/local/bin
+
+CONFIGURE_ARGS += CPP="/root/spksrc/toolchains/syno-apollolake-6.1/work/x86_64-pc-linux-gnu/bin/x86_64-pc-linux-gnu-cpp"
```

Comment:
Please don't use absolute links... use variables, since this will not work for anyone else!